### PR TITLE
transcript, information about players and window

### PR DESCRIPTION
**Pull request by rivalez**
transcript, information about players and window

### CODE REVIEW

File: battleship-server/src/test/java/com/academy/solid/nie/server/GameTest.java [hunk 1]
```diff
@@ -69,16 +69,4 @@ public void afterThirdInvokeOfMethodPlaySecondPlayerShouldReceiveFirstsPlayersMo
         verify(second, times(2)).inform(moveOfFirstPlayer);
```
Comment (EmiliaCiastek) [inline battleship-server/src/test/java/com/academy/solid/nie/server/GameTest.java]: create a local variable for '2'

File: battleship-client/src/main/java/com/academy/solid/nie/client/ui/Board.java [hunk 2]
```diff
@@ -119,15 +127,15 @@ private void markEndOfShip() {
...
-                Point2D.of(x + 1, y + 1)
+        Point2D[] array = new Point2D[] {
+            Point2D.of(x - 1, y),
+            Point2D.of(x + 1, y),
+            Point2D.of(x, y - 1),
+            Point2D.of(x, y + 1),
+            Point2D.of(x - 1, y - 1),
+            Point2D.of(x + 1, y - 1),
+            Point2D.of(x - 1, y + 1),
+            Point2D.of(x + 1, y + 1)
```
Comment (EmiliaCiastek) [inline battleship-client/src/main/java/com/academy/solid/nie/client/ui/Board.java]: maybe variable "step", not a lot of "+1" and "-1"

Comment (rivalez) [reply battleship-client/src/main/java/com/academy/solid/nie/client/ui/Board.java]: ok thanks

File: battleship-server/src/test/java/com/academy/solid/nie/server/GameTest.java (hunk 1 above)
Comment (rivalez) [reply battleship-server/src/test/java/com/academy/solid/nie/server/GameTest.java]: thanks

**Pull request merged**